back/spec/support/app_configuration_helper.rb

```diff
@@ -0,0 +1,15 @@
+module AppConfigurationHelper
+  def enable_phone_login
+    settings = AppConfiguration.instance.settings
+    settings.merge!(
+      'password_login' => {
+        'phone' => true,
+        'enabled' => true,
+        'allowed' => true,
+        'phone_email_pattern' => 'phone+__PHONE__@test.com'
+      }
+    )
+
+    AppConfiguration.instance.update(settings: settings)
```

Comment:
Also here I would prefer `update!`, but not blocking. Sorry for bothering you so much about these, the main reason I bring it up is that sometimes we get bugs or test failures that take a bit of time to debug before figuring out there's an `update` or `save` somewhere that is silently failing. Using `update!` and `save!` instead, makes it usually much easier to find out what's going wrong.

Reply:
True, I have the habit of not doing it but should change